internal/integration/integration_test.go

```diff
@@ -1132,6 +1139,7 @@ create table user_groups
 				err = printer.Fprint(&actualFields, token.NewFileSet(), fieldMethod)
 				r.NoError(err)
```

Comment:
i know its not from now but if you're touching this lets stop propagating the `require.Require` from the parent to the child test :)

Reply:
see line
https://github.com/ariga/entimport/pull/26/files#diff-75ef1fc3abbd5c7ab7baf68d3e4cd40c43acdac36451a42bd94cbfff773d40d2R1120

I actually fixed that

Reply:
oh cool <3